src/main/java/com/oltpbenchmark/api/config/Transaction.java

```diff
@@ -0,0 +1,7 @@
+package com.oltpbenchmark.api.config;
+
+public record Transaction(Class<? extends com.oltpbenchmark.api.Procedure> procedureClass,
```

Comment:
(unproductive comment) This is the first new Java feature in the wild that I've needed to look up. neat!